src/dbSta/src/dbNetwork.cc

```diff
@@ -144,6 +144,12 @@ ObjectId getDbNwkObjectId(dbObjectType typ, ObjectId db_id)
       return ((db_id << DBIDTAG_WIDTH) | DBMODULE_ID);
     } break;
     default:
+      logger_->error(
+          ORD,
+          2017,
+          "Error: unknown database type passed into unique id generation");
+      // note the default "exception undefined case" in database is 0.
+      // so we reasonably expect upstream tools to handle this.
```

Comment:
Is there a reason to not just remove the default? I think @QuantamHD is right, there is value in being a additionally expressive to avoid mistakes in the future.